docs/flow/modin/core/storage_formats/pyarrow/index.rst

```diff
@@ -0,0 +1,26 @@
+PyArrow storage format
+""""""""""""""""""""""
+
+.. toctree::
+    :hidden:
+
+    query_compiler
+    parsers
+
+In general, PyArrow storage formats follow the flow of the pandas ones: query compiler contains an instance of Modin Frame,
+which is internally split into partitions. The main difference is that partitions contain PyArrow tables,
+instead of DataFrames like with :doc:`pandas storage format </flow/modin/core/storage_formats/pandas/index>`. To learn more about this approach please
+visit :doc:`PyArrow execution engine </flow/modin/experimental/core/execution/ray/implementations/pyarrow_on_ray>` section.
```

Comment:
```suggestion
In general, PyArrow storage formats follow the flow of the pandas ones: query compiler contains an instance of Modin Dataframe,
which is internally split into partitions. The main difference is that partitions contain PyArrow tables,
instead of pandas DataFrames like with :doc:`pandas storage format </flow/modin/core/storage_formats/pandas/index>`. To learn more about this approach please
visit :doc:`PyArrowOnRay execution </flow/modin/experimental/core/execution/ray/implementations/pyarrow_on_ray>` section.
```